### DISCUSSION TITLE
Getting below error

### DISCUSSION THREAD

**Discussion by skand-tripathi** [open-telemetry/opentelemetry-java-instrumentation#6545] [Q&A]
java -javaagent:opentelemetry-javaagent.jar \
     -Dotel.resource.attributes=service.name=your-service-name \
     -Dotel.traces.exporter=zipkin \
     -jar target/*.jar

Error : [otel.javaagent 2022-09-05 19:53:31:021 +0530] [OkHttp http://localhost:4317/...] ERROR io.opentelemetry.exporter.internal.grpc.OkHttpGrpcExporter - Failed to export metrics. The request could not be executed. Full error message: Failed to connect to localhost/0:0:0:0:0:0:0:1:4317

**Comment by mateuszrzeszutek** [comment]
Hey @skand-tripathi,
The javaagent comes with the metrics exporter set to `otlp` by default. If the collector instance/backend you're sending telemetry to does not accept metrics data, you'll need to disable the metrics exporter: `-Dotel.metrics.exporter=none`
